### ISSUE TITLE
Duplicate import gql from 'graphql-tag'

### ISSUE THREAD

**Issue by polRk** [dotansimha/graphql-code-generator#3504]
Duplicate `import gql from 'graphql-tag'`
<img width="514" alt="Снимок экрана 2020-02-14 в 11 58 10 AM" src="https://user-images.githubusercontent.com/39828645/74516342-46e2b880-4f21-11ea-82bc-cd9994d31d75.png">
<img width="505" alt="Снимок экрана 2020-02-14 в 11 58 47 AM" src="https://user-images.githubusercontent.com/39828645/74516399-5d890f80-4f21-11ea-84a3-22df0f3627d5.png">

codegen.yml

```yaml


overwrite: true

documents: "src/**/*.graphql"

config:
  skipTypename: true
  avoidOptionals:
    inputValue: false
    object: true
  strict: true
  preResolveTypes: true
  scalars:
    timestamptz: string
    uuid: string
    jsonb: "{ [key: string]: any }"

generates:
  src/generate.ts:
    plugins:
      - add: '/* eslint:disable */'
      - "typescript"
      - "typescript-operations"
      - "typescript-document-nodes"
      - "typescript-graphql-request"
```

**Comment by dotansimha** [comment]
@polRk 

I pushed a fix for the duplications, can you please try the alpha? `1.12.3-alpha-c868a2f8.36` 

Thanks!

**Comment by dotansimha** [comment]
@polRk 

I pushed a fix for the duplications, can you please try the alpha? `1.12.3-alpha-ed40601c.37` 

Thanks!

**Comment by polRk** [comment]
@dotansimha, Yes i can, I'll do it next week.

**Comment by Aleksion** [comment]
Is it possible to NOT import gql?
"typescript-graphql-request" uses print to convert it to a string anyways, so it seems like an unnecessary addition to the bundle size..

**Comment by dotansimha** [comment]
Fixed in `v1.13.0`

@Aleksion you can either use `documentNode` or `noGraphQlTag` option for that.
